From:    Wolfram Sang <w.sang@pengutronix.de>
Subject: [Quilt-dev] [patch 2/3] import.test: always have a local patches-dir
Date:    Sat, 11 Oct 2008 15:47:22 +0200
To:      quilt-dev@nongnu.org
cc:      

The test for import did not always recreate a patches-dir inside the temporary
location, so it would fail if you modified quilt using quilt.

Signed-off-by: Wolfram Sang <w.sang@pengutronix.de>
---
 test/import.test |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

Index: test/import.test
===================================================================
--- test/import.test.orig	2008-11-05 14:14:37.911126000 -0600
+++ test/import.test	2008-11-05 14:14:41.151317000 -0600
@@ -229,7 +229,8 @@
 	> No patches applied
 
 	# Test importing multiple patches at once
-	$ rm -rf .pc patches
+	$ rm -rf patches/ .pc/
+	$ mkdir patches
 	$ quilt import t/patch1.diff t/patchR.diff
 	> Importing patch t/patch1.diff (stored as patches/patch1.diff)
 	> Importing patch t/patchR.diff (stored as patches/patchR.diff)
@@ -238,7 +239,8 @@
 	> patch1.diff
 	> patchR.diff
 
-	$ rm -rf .pc patches
+	$ rm -rf patches/ .pc/
+	$ mkdir patches
 	$ quilt import t/patchR.diff
 	> Importing patch t/patchR.diff (stored as patches/patchR.diff)
 	$ quilt import t/patch1.diff
